package/MDAnalysis/coordinates/base.py

```diff
@@ -137,6 +140,28 @@
 from . import core
 from .. import NoDataError
 
+from ..auxiliary.base import AuxReader
+from ..auxiliary.core import get_auxreader_for
+
+class Namespace(object):
```

Comment:
I guess this would live better in the lib/util module since it is a utility class